pom.xml

```diff
@@ -170,6 +170,9 @@
             <plugin>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.21.0</version>
+                <configuration>
+                    <useSystemClassLoader>false</useSystemClassLoader>
```

Comment:
A few days ago I updated my linux packages (jdk8 was on the list). Then compilation fails. I read that it because Debian package bug (https://stackoverflow.com/questions/53010200/maven-surefire-could-not-find-forkedbooter-class). In the StackOverflow I found solution to insert that <configuration> and all works fine now for me.

Reply:
I hope it won't affect your teammates